### ISSUE TITLE
review of example that Arno shared 

### ISSUE THREAD

**Issue by robertoostenveld** [bids-standard/bep021#9]
On https://drive.google.com/drive/folders/1UepEeZ80wpPxU-OhmiR9oGphomwRR2oX @arnodelorme shared an example derivative EEG dataset. This issue serves to share comments on that draft dataset.

**Comment by robertoostenveld** [comment]
also tagging @dungscout96 who was involved in the construction of the example.

**Comment by robertoostenveld** [comment]
the directory structure is 

```
ds004117der
├── ds004117der
│   ├── sub-001
│   │   └── ses-01
│   │       └── eeg
│   ├── sub-002
│   │   └── ses-01
│   │       └── eeg
```

where I guess the top-level directory is the raw, and the folder inside it is the derivative. However, there should be a `derivative` folder level in between like this

```
ds004117raw
├── derivatives
|   └── ds004117der
```
See https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#source-vs-raw-vs-derived-data

**Comment by robertoostenveld** [comment]
according to https://bids-specification.readthedocs.io/en/stable/05-derivatives/01-introduction.html#file-naming-conventions I think the derivative data should be distinguished from the raw data by a `desc-<label>` entity.

**Comment by robertoostenveld** [comment]
there is no `descriptions.tsv` file as discussed in #8

**Comment by robertoostenveld** [comment]
Regarding the directory structure, I am not sure whether my assumption at https://github.com/bids-standard/bep021/issues/9#issuecomment-1725542841 holds. It rather seems that there is a copy of the same directory in the directory, and that both are derived. The metadata seems to be exactly the same, including the `GeneratedBy` in the dataset_description.json

**Comment by arnodelorme** [comment]
The folder in the subfolder was an issue associated with copying the data to Google Drive. I have removed the subfolder. I was under the impression that the inclusion of the raw data was not mandatory.

**Comment by arnodelorme** [comment]
I have updated the data (see the link above) - I have also removed the data itself to make it lightweight.

**Comment by effigies** [comment]
> the inclusion of the raw data was not mandatory.

Correct. If the dataset has `"DatasetType": "derivative"` in its `dataset_description.json`, then it would be recognized as a standalone derivative dataset.
